woocommerce/woocommerce-android
# [Mobile Payments] Track error cases of the plugin installation API call

Closes: #9712


### Description

The PR adds tracking of the `CARD_PRESENT_ONBOARDING_CTA_FAILED` event with `reason` and `error_description`

### Testing instructions

* Delete WCPay from your site
* Go to Menu -> Payments
* Notice an error at the bottom and click on it
* Simulate the error either with disabling internet or using proxyman (keep in mind that with disabled internet connection it can take while as for some reason we retry calls)
* In the logs, notice `CARD_PRESENT_ONBOARDING_CTA_FAILED` tracked

```
card_present_onboarding_cta_failed, Properties: {"reason":"plugin_install_tapped","error_description":"An error occurred","plugin_slug":"woocommerce-payments","country":"unknown","blog_id":198219640,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true}
```


### Images/gif
